cpp/lib/ome/bioformats/tiff/IFD.cpp

```diff
@@ -1057,7 +1090,11 @@ namespace ome
             break;
 
           default:
-            throw Exception("Unsupported OME data model PixelType");
+            {
+              boost::format fmt("Unsupported OME data model PixelType %1%");
```

Comment:
I don't think so here.  It's the same as most of the messages above: the TIFF IFD does not support a given PixelType; it's also never going to be seen since we support every pixel type in the data model (at present); it's here only to catch any future model changes.